### PR TITLE
refactor: `comment-parser` approach for default tags

### PR DESCRIPTION
**Pull request by brettz9**
refactor: use `comment-parser` approach to avoid special type processing for default tag

Note that this doesn't undo the recent special processing. You'll have to figure out how you want to deal with it, whether to revert the changes entirely or not.

### CODE REVIEW

File: src/parser.ts [hunk 1]
```diff
@@ -72,6 +79,18 @@ export const getParser = (originalParse: Parser["parse"], parserName: string) =>
 
       const parsed = parse(commentString, {
         spacing: "preserve",
+        tokenizers: [
+          tagTokenizer(),
+          (spec) => {
+            if ([DEFAULT, DEFAULT_VALUE].includes(spec.tag)) {
```
Comment (Dan503) [inline src/parser.ts:85]: We can use the `isDefaultTag` util here instead.

Comment (hosseinmd) [reply src/parser.ts:85]: I will merge this, we could do this in another PR

Comment (Dan503) [reply src/parser.ts:85]: Do the tests still pass?

Comment (hosseinmd) [reply src/parser.ts:85]: yes Passed

**Comment by brettz9** [comment]
Might I suggest closing https://github.com/syavorsky/comment-parser/issues/155#issuecomment-1100642428 then if this is ok?

**Pull request merged**